gtests/net/mptcp/sockopts/test_unsupported_sockopts_tcp_inq.pkt

```diff
@@ -0,0 +1,7 @@
+-tolerance_usecs=100000
+`../../common/defaults.sh`
+
+// test 1 : Unsupported options
+0.0   socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0.0  setsockopt(3, SOL_TCP, TCP_INQ, [1], 4) = -1 EOPNOTSUPP (Operation not supported)
++0.0  getsockopt(3, SOL_TCP, TCP_INQ, [0], [4]) = -1 EOPNOTSUPP (Operation not supported)
```

Comment:
I am sorry, I am not sure I understand. Do we include the TCP_INQ one in another commit?

Reply:
My bad, I wanted to say: `s/check/keep`.

In your commit message, you mention you add support for `IP_TOS`  but here this file is not about `IP_TOS` but `TCP_INQ`. Best to have one commit dedicated to `IP_TOS`. If you want to support `TCP_INQ`, please do it in a separated commit.

Reply:
Ah, I understand. Maybe I should edit my commit message to say that there are a bunch of tests and not just to check IP_TOS. Or would it be better to have the TCP_INQ testin another commit?

Reply:
After this is resolved, I'll squash the commits and push the changes all together.

Reply:
@mattbe I am not entirely sure what to do. I have an idea -
1. I put one test (let's say the one in mptcp/sockopts) in a new commit and squash all my previous commits together with this one. But in this case, I would still have the old files(mptcp_unsupported) present I think? And then do another commit for mptcp_unsupported.

OR

2. I could do a git reset to the upstream head and start the process all over.

wdyt would be a better idea?

Reply:
Up to you but from what I see on this PR, it seems needed to rewrite the commits. If you prefer to "reset and commit" or "rebase", that's the same. If you are not used to manipulate Git, it might look easier to use:

```
git reset --soft origin/mptcp-net-next
git restore --staged .
git add -p
git commit -s
```

But you can always try to do: `git rebase -i origin/mptcp-net-next` and follow the instructions. Maybe best to create a "backup" branch (`git branch backup`) before doing that so you can easily find the previous version you had before (even if `git status` will also tell you to use `git rebase --abort` to go back to the previous state).